File: back/src/main/java/org/example/anibuddy/chat/dto/ChatRoomResponse.java
package org.example.anibuddy.chat.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;

@AllArgsConstructor
@Setter
@Getter
public class ChatRoomResponse {

    private int roomId;

    private int myId;
    private String myRole;
    private String myName;
    private String myImageUrl;

    private int otherId;
    private String otherRole;
    private String otherName;
    private String otherImageUrl;
}

File: front/app/src/main/java/com/example/front/data/response/ChatRoomResponse.kt
package com.example.front.data.response

data class ChatRoomResponse(
    val roomId: Int,
    val myName: String,
    val myId: Int,
    val myRole: String,
    val myImageUrl: String,
    val otherName: String,
    val otherImageUrl: String,
    val otherId: Int,
    val otherRole: String
)
